doc/administrator_guide.md

```diff
@@ -94,6 +94,28 @@ Select the account key for use with Google Cloud API (which kcidb uses):
 Clone kcidb repository, checkout the revision you want to deploy, and install
 kcidb as described in [README.md](README.md).
 
+### Additional Dependencies
+
+While deploying you may also need to install psql. psql is a command-line tool for interacting with the PostgreSQL database. It is used to manage and manipulate data stored in a PostgreSQL database, as well as to administer and configure the database itself. 
+
+Update the package lists:
+    
+    sudo apt-get update
+
```

Comment:
We don't need to describe this step, our users generally know how to update the package list for their distribution.